languages/ruby/exercises/concept/floating-point-numbers/.docs/after.md

```diff
@@ -0,0 +1,12 @@
+A floating-point number is a number with zero or more digits behind the decimal separator. Examples are `4.0`, `0.1`, `3.14`, `-6.4` `16.984025` and `1024.0`. In Ruby, floating-point numbers are implemented through the [Float](https://ruby-doc.org/core-2.7.0/Float.html) class.
+
+You can find a short introduction to floating-point numbers at [0.30000000000000004.com][0.30000000000000004.com].
+
+The [Float Toy page][evanw.github.io-float-toy] has a nice, graphical explanation how a floating-point numbers' bits are converted to an actual floating-point value.
+
+To repeatedly execute logic, one can use loops. In this example the `while` loop is useful because it keeps on looping _while_ a boolean condition evaluates to `true`. Ruby implements a loop similar to the `while` loop. It's called the `until` loop, and you've probably guessed what it does. It keeps looping _until_ a boolean condition evaluates to `true`. In some languages, to make a piece of code execute an unlimited number of times, constructs like `while true` are used. In Ruby, the `loop` loop exists for that purpose. Even though the `loop` loop does not depend on a single condition, it can be canceled by using a `return` or `break` keyword.
```

Comment:
Could you add code samples that show the syntax of the `while`, `until` and `loop` constructs, as this will be the first time the student is introduced to their syntax, is it?

Reply:
Yeah, it is. I will add an example. That makes me wonder if the `floating-point-numbers` name for the exercise is the most fitting. Should `loops` be added to the exercise name maybe?